lms/djangoapps/sneakpeek_deeplink/middleware.py

```diff
@@ -19,6 +22,8 @@
     'info',
 )
 
+log = logging.getLogger("edx.sneakpeek_middleware")
```

Comment:
Actually, if pylint takes issue w/ this syntax,
the proper course of action is to actually follow
the advised casing, instead of disabling the warning.

The benefit to keeping constants in uppercase is that
it visually flags that this value should never change.
The linter can then flag if we attempt to reassign the value.

By disabling the warning, we lose all of these benefits.